asciidoc/modules/con_mg_changes-in-json.adoc

```diff
@@ -107,7 +107,25 @@ If you want to continue using the Base64 encoder in {VertX} {v4}, you can use th
 java -Dvertx.json.base64=legacy ...
 ----
 
-A new utility method `Json.toBase64()` converts the Base64URL string to Base64. This utility can be for:
+Users that must use both {VertX} {v3x} and {VertX} {v4} releases may need to create some utility to convert from Base64
+to Base64URL. Given that both encodings are Base64, we're just changing the alphabet which makes this a trivial task:
```

Comment:
@pmlopes Did I get the context right here?

If you want to use both {VertX} {v3x} and {VertX} {v4} releases, you can use the following utility to convert the Base64
string to Base64URL. Since both the encodings are Base64, you will have to make minimal updates to your existing applications.